deploy/chart/catalog_resources/rh-operators/descheduler.csv.yaml

```diff
@@ -0,0 +1,142 @@
+      or removed from nodes, pod/node affinity requirements are not satisfied any more.
+      * Some nodes failed and their pods moved to other nodes.
+       New nodes are added to clusters.
+
+      Consequently, there might be several pods scheduled on less desired nodes in a cluster.
+      Descheduler, based on its policy, finds pods that can be moved and evicts them. Please
+      note, in current implementation, descheduler does not schedule replacement of evicted pods
+      but relies on the default scheduler for that.
+
+      ## Compatibility matrix
+
+          Descheduler  | supported Kubernetes version
+          -------------|-----------------------------
+          0.4          | 1.9+
+          0.1-0.3      | 1.7-1.8
+
+      ## Note
+
+      This project is under active development, and is not intended for production use.
+      Any api could be changed any time with out any notice. That said, your feedback is
+      very important and appreciated to make this project more stable and useful.
+    repository: https://github.com/openshift/descheduler-operator
+    support: Red Hat
+  name: descheduler.v0.0.1
+  namespace: openshift-descheduler-operator
+spec:
+  customresourcedefinitions:
+    owned:
+    - description: Represents an instance of a Descheduler application
+      displayName: Descheduler Operator
+      kind: Descheduler
+      name: deschedulers.descheduler.io
+      version: v1alpha1
+  displayName: Descheduler
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - pods
+          - configmaps
+          - secrets
+          - names
+          - nodes
+          - pods/eviction
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          verbs:
+          - '*'
+        - apiGroups:
+          - batch
+          - extensions
+          resources:
+          - jobs
+          verbs:
+          - '*'
+        - apiGroups:
+          - descheduler.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: openshift-descheduler
+      deployments:
+      - name: descheduler-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: descheduler-operator
+          template:
+            metadata:
+              labels:
+                app: descheduler-operator
+            spec:
+              containers:
+              - command:
+                - descheduler-operator
+                env:
+                - name: MY_POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: MY_POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
```

Comment:
This looks like the operator only watches the namespace it runs it, but it requires cluster-level permissions for its service account.